project/src/components/app/app.tsx

```diff
@@ -7,7 +16,35 @@ type AppProps = {
 
 function App({offersCount, displayedOffersCount}: AppProps): JSX.Element {
   return (
-    <MainPage offersCount={offersCount} displayedOffersCount={displayedOffersCount} />
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path={AppRoute.Root}
+          element={<MainPage offersCount={offersCount} displayedOffersCount={displayedOffersCount} />}
+        />
+        <Route
+          path={AppRoute.Login}
+          element={<Login />}
+        />
+        <Route
+          path={AppRoute.Favorites}
+          element={
+            <PrivateRoute authorizationStatus={AuthorizationStatus.NoAuth}>
+              <Favorites />
+            </PrivateRoute>
+          }
```

Comment:
```suggestion
          )}
```

многострочку лучше в круглые скобки засовывать, когда они используются как выражение